components/butterfly-test/src/lib.rs

```diff
@@ -191,14 +217,29 @@ impl SwimNet {
     }
 
     pub fn health_of(&self, from_entry: usize, to_entry: usize) -> Option<Health> {
+        assert!(cfg!(feature = "deadlock_detection"),
```

Comment:
shouldn't this be `cfg!(not(feature = "deadlock_detection"))`? I'm getting `This test should be run with --features=deadlock_detection` even if I specify it in cargo test.